src/custom-elements/src/constants/color.ts

```diff
@@ -1,3 +1,7 @@
 export const TEXT_COLORS = ['', 'primary', 'success', 'warning', 'danger', 'muted', 'inverse'];
 
 export const SEMANTIC_COLORS = ['info', 'warning', 'danger', 'success'];
```

Comment:
@janibolkvadze All this consts are suitable to be defined `as const` as you did for `ICON_COLORS`. They are read-only arrays of strings.

Reply:
I will do one big refactoring of all the existing constant exports and improve valid types of current web components in another PR ;)